how-to/business-logic/how-to-change-user-role.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to Change User Role Programatically
+description: An example on changing the user role programatically, using cloud code
+type: how-to
+page_title: How to Change User Role Programatically
+slug: bl-change-user-role
+position:
+tags: backendservices, business-logic, cloud-code, user-role, cloud-functions
+teampulseid: 
+ticketid: 1016776
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+Frequently you need to set programmatically the role of a user account based on certain logic. For example, when the user achieves certain rank or performs an in-app purchase. This requires administrative privileges for the app and can be achieved with server-side logic.
+
+## Solution
+
+You can add such logic to the business logic layer - for example you may expose a Cloud Function available to administrattors only or after a user account is created - check for certain conditions and update the role.
+
+Updating a user role requires the use of a [Master Key]({http://docs.telerik.com/platform/backend-services/javascript/security/security-master-key-auth}) - it is strongly recommended not to include your Master Key inside your app code - therefore a perfect solution is to use clode code as the code stays on {{site.tp}} server. 
+
+A sample function in cloud code to change the user role would be:

```

Comment:
Here is an example using the JavaScript SDK in the Cloud Code to change the user role: